python/cudnn/gemm/cutedsl/discrete_grouped/dswiglu/jax_api.py

```
@@ -0,0 +1,399 @@
+        "k",
+        "k",
+        "n",
+        BlockScaledDiscreteWeightDgluDbiasGroupedGemmKernel.FIX_PAD_SIZE,
+        act_func,
+    ):
+        raise ValueError("Unsupported discrete grouped GEMM dSwiGLU tile, cluster, alignment, or layout configuration")
+
+    # SFD is generated only for the fp8-in/fp8-out E8M0 configuration; 16-bit d
+    # accumulates per-expert amax instead (both mirror the eager wrapper).
+    has_amax = d_dtype in _amax_dtypes
+    if not (d_dtype in _fp8_dtypes) and discrete_col_sfd:
```

Ruff (0.16.1)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)